MekHQ/src/mekhq/campaign/mission/resupplyAndCaches/PerformResupply.java

```diff
@@ -177,12 +178,12 @@ public static void performResupply(Resupply resupply, AtBContract contract, int
             }
         }
 
-        logger.info("totalTonnage: " + totalTonnage);
+        logger.info(String.format("totalTonnage: %s", totalTonnage));
```

Comment:
Logger .info should support VARRAGs so don't need to do this. Just pass them directly like you would String.format.